2020/index.html

```diff
@@ -114,14 +114,17 @@
                       <li class="rd-nav-item active">
                         <a class="rd-nav-link" href="#">Home</a>
                       </li>
+
                       <li class="rd-nav-item">
                         <a class="rd-nav-link" href="#speakers">Speakers</a>
                       </li>
-                      <!-- <li class="rd-nav-item">
-                        <a class="rd-nav-link" href="#event-schedule"
-                          >Schedule</a
+
```

Comment:
@sahilbhatia we need to update this index.html which is in base folder. remove this file to avoid confusion. Also lets use josh's black font logo for consistency.